docs/src/main/sphinx/develop/header-based-authenticator.rst

```diff
@@ -0,0 +1,45 @@
+==========================
```

Comment:
Please mention how to setup multiple authenticators with `header-based-authenticator.config-files`. Also please mention what configuration has to be placed in `config.properties` to make it work end to end.